### DISCUSSION TITLE
Use rodauth with specific host

### DISCUSSION THREAD

**Discussion by me-kirill** [janko/rodauth-rails#218] [Q&A] [answered]
Hi!

I have two hostnames:
- one for admin (admin.app.com)
- second for client (app.com)

I need to use authentication (routes etc) only on admin.app.com

How can I set up this?

**Comment by janko** [comment, marked as answer]
The `r.rodauth` call in you Rodauth app's `route` block is what routes Rodauth endpoints, so you should be able to just wrap a conditional around that code that checks for the hostname through the Roda request object.

```rb
# app/misc/rodauth_app.rb
class RodauthApp < Rodauth::Rails::App
  route do |r|
    # ...
    if request.host == "admin.app.com"
      r.rodauth
    end
    # ...
  end
end
```

**Reply by me-kirill** [reply]
Thank you!
